core/domain/exp_domain.py

```diff
@@ -1186,7 +1173,8 @@ def create_default_interaction(cls, default_dest_state_name):
         return cls(
             cls._DEFAULT_INTERACTION_ID,
             {}, [],
-            Outcome(default_dest_state_name, [], {}), [], [], {}
+            Outcome(
+                default_dest_state_name, SubtitledHtml('', {}), {}), [], [], {}
```

Comment:
Can we use something like SubtitledHtml.create_default() here instead?

Reply:
done